fe/fe-core/src/test/resources/sql/tpch-histogram-cost/q1.sql

```diff
@@ -0,0 +1,146 @@
+[sql]
```

Comment:
I think we needn't keep all 22 sql files.

Reply:
Because I have tested 22 sqls at present, all of them have plan changes and involve the corresponding histogram function, so I added them to UT